packages/0x.js/src/contract_wrappers/ether_token_wrapper.ts

```diff
@@ -163,6 +163,15 @@ export class EtherTokenWrapper extends ContractWrapper {
     public unsubscribeAll(): void {
         super.unsubscribeAll();
     }
+    /**
+     * Retrieves the Ethereum address of the EtherToken contract deployed on the network
+     * that the user-passed web3 provider is connected to. If it's an unknown private network - undefined is returned.
+     * @returns The Ethereum address of the EtherToken contract or undefined.
+     */
+    public getContractAddressIfExists(): string | undefined {
```

Comment:
I know that `getContractAddressIfExists` is more descriptive than `getContractAddress` in this case, but I also feel like the name should be standardized for all of our wrappers. Thoughts?

Reply:
I prefer descriptive over standardised. It makes this much more readable for the next developer.